lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -166,6 +166,53 @@ def self.zero_pad_macaddr(mac)
       sprintf('%04x.%04x.%04x', o1, o2, o3)
     end
 
+    # This will expand the string to a list of bds as integers
+    def self.string_to_array(string)
```

Comment:
I need these two methods string_to_array and unsorted_list_to_array because they are extensively used in bridge_domain_vni for data manipulation.